okpt/io/config/parser.py

```diff
@@ -0,0 +1,31 @@
+"""Provides methods to parse YAML and JSON files."""
+
+import json
+from io import TextIOWrapper
+from typing import Any, Dict
+
+import yaml
+
+from okpt.io import reader
+
+
+def parse_yaml(file: TextIOWrapper) -> Dict[str, Any]:
+    """Parses YAML file from file object."""
```

Comment:
Follow this convention for file doc string comments: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods.

Reply:
For some of the shorter functions with only a short description, I just followed 

>A function must have a docstring, unless it meets all of the following criteria:
>  * not externally visible
> * very short
> * obvious

as stated in that section.

Reply:
So we will be following this guideline: https://google.github.io/styleguide/pyguide.html

Reply:
@VijayanB yes. Do you recommend something else?

Reply:
Also, isnt this function externally visible?

Reply:
Yeah it is externally visible, I'll add more to the docstring then.

Reply:
@jmazanec15 No. google python guide is good. Just to confirm we will be following this end to end.

Reply:
Done, I've updated some other functions as well.